web/src/main/java/nl/tehdreamteam/se42/web/soap/user/SoapUserEndpoint.java

```diff
@@ -43,6 +43,11 @@ public String login(@WebParam(name = "username") String username, @WebParam(name
     }
 
     @Override
+    public long registerUser(String username, String password) {
+        return 0;
+    }
+
+    @Override
     public List<Long> getConversations(String token) {
         return Arrays.asList(1L, 2L, 4L);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 2L to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS109)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 4L to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS109)